packages/dev-approuter/lib/helpers.js

```diff
@@ -0,0 +1,133 @@
+const path = require("path");
+
+/**
+ * Parses the approuter configuration from an `xs-dev.json` file.
+ * If that file doesn't exist, it looks for an `xs-app.json` file.
+ * Also scans the routes of the config file and checks for the `dependency` property
+ * and stores those in `config.dependencyRoutes`, which is where the dev approuter will do its magic.
+ * @returns {Object} the approuter configuration including all `dependencyRoutes`.
+ */
+const parseConfig = () => {
+    let config;
+    let configFile;
+    try {
+        config = require(path.join(process.cwd(), "xs-dev.json"));
+        configFile = "xs-dev.json";
```

Comment:
can be moved above line 14 and then you can reuse the `configFile` for the `require` call. But honestly, I would work with the `fs` module and check whether the file exists `fs.existsSync` and then read it via `JSON.parse(fs.readFileSync(path.join(process.cwd(), configFile, { encoding: "utf8" }))`.

Reply:
Yep, ok